man/systemd-sleep.conf.xml

```diff
@@ -78,9 +78,20 @@
         <term>suspend-then-hibernate</term>
 
         <listitem><para>A low power state where the system is initially suspended
-        (the state is stored in RAM). If not interrupted within the delay specified by
-        <command>HibernateDelaySec=</command>, the system will be woken using an RTC
-        alarm and hibernated (the state is then stored on disk).
+	(the state is stored in RAM) and then hibernated based on battery percentage
+        capacity. If the current battery capacity is higher than 5%, the system goes
+        back to suspend for interval calculated using battery disharge rate per hour.
+        Battery discharge rate per hour is stored in
+        <filename>/var/lib/systemd/sleep/battery_discharge_percentage_rate_per_hour</filename>
```

Comment:
Don't mention this path, that's an internal design decision, and we should be free to change it any time. Just mention it is stored, without saying where precisely.

i.e. document user-facing behaviour. Do not document internal implementation details in user-facing docs

Reply:
true.  fixed

Reply:
Not fixed? You pushed an old version??